test/test_add.cc

```diff
@@ -355,10 +171,10 @@ void test_add_work(Params& params, bool run)
             // Do this on full m-by-n matrix to detect if on, say,
             // a lower triangular matrix, the kernel accidentally modifies
             // the upper triangle.
-            slate::add( -one, Aref_full, one, Afull );
-            slate::add( -one, Bref_full, one, Bfull );
-            real_t A_diff_norm = slate::norm( slate::Norm::Max, Afull );
-            real_t B_diff_norm = slate::norm( slate::Norm::Max, Bfull );
+            subtract_matrices( Afull, Aref_full );
+            subtract_matrices( Bfull, Bref_full );
+            real_t A_diff_norm = slate::norm( slate::Norm::One, Aref_full );
+            real_t B_diff_norm = slate::norm( slate::Norm::One, Bref_full );
```

Comment:
Since this is an element-wise operation, I think it should be the max norm, but the `n` in the error equation below is probably wrong. Ideally, it would actually be
```
    error = max_ij( | A_ij – Aref_ij | / | Aref_ij | ),
```
rather than
```
    error = max_ij( | A_ij – Aref_ij | ) / max_ij( | Aref_ij | ).
```

Reply:
You're right.  I must have copied it without looking carefully at the norms.